dace/sdfg/graph.py

```diff
@@ -701,10 +701,10 @@ def remove_edge(self, edge: Edge[EdgeT]):
         del self._edges[t]
 
     def in_degree(self, node):
-        return len(self._nodes[node][0])
+        return self._nx.in_degree(node)
```

Comment:
Since which nx version are these methods available? We should also update the requirements.

Reply:
I checked, the methods have existed since at least 1.11, so it's fine :)
https://networkx.org/documentation/networkx-1.11/reference/generated/networkx.DiGraph.in_degree.html?highlight=in_degree#networkx.DiGraph.in_degree